pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -128,6 +130,33 @@ func (server *VerifierImpl) SendNonceRequest(ctx *zattest.Context) error {
 	return nil
 }
 
+//encodeVersions fetches EVE, UEFI versions
+func encodeVersions(quoteMsg *attest.ZAttestQuote) error {
+	quoteMsg.Versions = make([]*attest.AttestVersionInfo, 0)
+	eveVersion := new(attest.AttestVersionInfo)
+	eveVersion.VersionType = attest.AttestVersionType_ATTEST_VERSION_TYPE_EVE
+	eveRelease, err := ioutil.ReadFile(types.EveVersionFile)
+	if err != nil {
+		return err
+	}
+	eveVersion.Version = strings.TrimSpace(string(eveRelease))
+	quoteMsg.Versions = append(quoteMsg.Versions, eveVersion)
+
+	//GetDeviceBios returns empty values on ARM64, check for them
+	bVendor, bVersion, bReleaseDate := hardware.GetDeviceBios(log)
+	biosVendor := strings.TrimSpace(bVendor)
+	biosVersion := strings.TrimSpace(bVersion)
+	biosReleaseDate := strings.TrimSpace(bReleaseDate)
+	if biosVendor != "" && biosVersion != "" && biosReleaseDate != "" {
```

Comment:
I'm concerned we might be legitimate bios which sets version or releasedate but not both.
Can we make this check if any of vendor, version, date are set and report in that case, instead of requiring all to be set?